app/assets/stylesheets/modules/global-alert.scss

```diff
@@ -1,7 +1,16 @@
- $global-alert-color: rgb(204, 0, 0);
+$global-alert-color: rgb(204, 0, 0);
+$su-bg-digital-green: rgb(0, 155, 118);
```

Comment:
It looks like we have all the colors in https://github.com/sul-dlss/mylibrary/blob/main/app/assets/stylesheets/sul-variables.scss  Would it make sense to move these there?

Reply:
Yes, it'd be nice to have all the SUL colors in one place. I noticed those and thought that perhaps they are out-of-date? For example, "mint" (#009b76) appears to correspond to Digital Green (#008566). But, white text on "mint" is not WCAG AA compliant.